app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -57,6 +58,7 @@ class ToolDetailsFragment() : BasePlatformFragment<ToolDetailsFragmentBinding>(R
         super.onBindingCreated(binding, savedInstanceState)
         binding.fragment = this
         binding.tool = dataModel.tool
+        binding.hasTraining = dataModel.primaryManifest.map { !it?.tips.isNullOrEmpty() }
```

Comment:
For now only enable training for debug builds
```suggestion
        binding.hasTraining = dataModel.primaryManifest.map { !it?.tips.isNullOrEmpty() && BuildConfig.DEBUG }
```